README.md

```diff
@@ -1 +1,53 @@
 # VGG Web
+
+![VGG hero image](https://docs.verygoodgraphics.com/img/vgg_docs_bg.jpg)
+
+A JavaScript/TypeScript/React/Vue/Svelte and Web Assembly ([WASM](https://developer.mozilla.org/en-US/docs/WebAssembly)) runtime library for [VGG](https://vgg.cool).
```

Comment:
Consider changing all the short URLs `https://vgg.cool` to the canonical URL `https://verygoodgraphics.com/` in case of redirect failure.